SoftLayer/CLI/event_log/get.py

```diff
@@ -59,9 +60,11 @@ def cli(env, date_min, date_max, obj_event, obj_id, obj_type, utc_offset, metada
             if username is None:
                 try:
                     username = user_mgr.get_user(log['userId'], "mask[username]")['username']
-                    user_data[log['userId']] = username
+                # Some users might not be able to access information about this specific userId
```

Comment:
Maybe to avoid making this request, we can get the username using this mask
mask = "mask[eventName,label,objectName,eventCreateDate,userId,userType,objectId,metaData,user[username]]"

In the request of the logs
https://github.com/softlayer/softlayer-python/blob/master/SoftLayer/CLI/event_log/get.py#L41

Reply:
In this case, adding a "deep" property like username to all the events we get back from the API can be pretty expensive and might make the command timeout sometimes.  I think getting each user after the fact may take a little more time, but it helps prevent the ` event_mgr.get_event_logs(request_filter)` from timeing out